Directory.Packages.props

```
@@ -54,7 +54,7 @@
     <PackageVersion Include="Microsoft.TestPlatform.TranslationLayer" Version="$(MicrosoftNETTestSdkVersion)" />
     <PackageVersion Include="Microsoft.Win32.Registry" Version="5.0.0" />
     <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
-    <PackageVersion Include="Polyfill" Version="9.0.0-beta.4" />
+    <PackageVersion Include="Polyfill" Version="9.0.0-beta.7" />
```

I just started a mirror of latest version. It may take 30mins to complete.

```suggestion
    <PackageVersion Include="Polyfill" Version="9.0.0-beta.9" />
```